modules/tensor_mechanics/test/tests/shell/static/plate_bending.i

```diff
@@ -0,0 +1,199 @@
+# Test for simply supported plate under uniform pressure
+
+# One quarter of a 50 m x 50 m x 1m plate is modeled in this test.
+# Pressure loading is applied on the top surface using nodal forces
+# of magnitude -10 N on all nodes. This corresponds to a pressure (q) of
+# -10.816 N/m^2.
+
+# The FEM solution at (0,0), which is at the center of the full plate
+# is -2.997084e-03 m.
+
+# The analytical solution for displacement at center of plate obtained
+# using a thin plate assumption for a square plate is
+# w = 16 q a^4/(D*pi^6) \sum_{m = 1,3,5, ..}^\inf \sum_{n = 1,3,5, ..}^\inf  (-1)^{(m+n-2)/2}/(mn*(m^2+n^2)^2)
+
+# where a = 50 m, q = -10.816 N/m^2 and D = E/(12(1-v^2))
+# The analytical series solution converges to 2.998535904e-03 m
+# when the first 16 terms of the series are considered (i.e., until
+# m & n = 7).
+
+# The resulting relative error between FEM and analytical solution is
+# 0.048%.
```

Comment:
None of the tests output stresses or strains. Can you add those just to see what they look like? Does it make sense for any of these tests to compare stress or strain to analytical solutions as well?

Reply:
I have added the capability to compute the stresses/strains in the global coordinate system. This capability is tested in the beam bending test. 

Also, I have added a two other tests, a rotated cantilever beam with the beam in the XZ plane and a pinched cylinder problem to test the element in other coordinate systems.